doc/strategies/LimitOrder.md

```diff
@@ -0,0 +1,28 @@
+# LimitOrder
+
+The `LimitOrder` contract provides order book style trading functionalities which is similar to what centralized exchange does. Orders can be queried from the off-chain order book system and then be filled on chain. Also, it supports order cancelling and partially fill. The `LimitOrder` contract shares the `AllowanceTarget` design from Tokenlon which allow users to create an order without depositing tokens first. Tokens are transferred between taker and maker only when the order gets filled on chain. This design provides the certainty of price for maker while maintaining filling flexibilities for taker at the same time.
+
+Besides the traditional taker scenario, an order can also be satisfied by liquidity from a supported AMM protocol. It relies on someone(EOA) trying to match the price of AMM protocols and the opened orders. In order to incentivize relayers to participate, the difference of the price is considered as the profit of a relayer.
```

Comment:
```suggestion
Besides the traditional taker scenario, an order can also be satisfied by liquidity from a supported AMM protocol. It relies on relayer(EOA) to match the price of AMM protocols and the opened orders. In order to incentivize relayers to participate, the difference of the price is considered as the profit of a relayer.
```

Reply:
`Relayer` may cause reader thinks that there're only a group of accounts are able to do that. So I used `someone` to emphasize everyone can participate.

Reply:
`relayer` is mentioned in the next paragraph: `... In order to incentivize relayers to participate, the difference of the price is considered as the profit of a relayer.`
If it is not mentioned here then readers won't know to what does the `relayer` refer.